scripts/biomedical/chembl/tmcfs/chembl_uniprot.tmcf

```diff
@@ -0,0 +1,7 @@
+Node: E:chembl_uniprot->E1
+typeOf: dcs:Protein
+dcid: C:chembl_uniprot->dcid
+name: C:chembl_uniprot->Name
```

Comment:
Please confirm that the name properties won't be colliding across all of these files. Each name should be unique to each dcid and should ideally be human understandable.